pkg/protocol/xprotocol/dubbo/command.go

```diff
@@ -0,0 +1,104 @@
+package dubbo
+
+import (
+	"mosn.io/mosn/pkg/protocol/xprotocol"
+	"mosn.io/mosn/pkg/types"
+)
+
+type Header struct {
+	Magic   []byte
+	Flag    byte
+	Status  byte
+	Id      uint64
+	DataLen uint32
+
+	Event           int // 1 mean ping
+	TwoWay          int // 1 mean req & resp pair
+	Direction       int // 1 mean req
+	SerializationId int // 2 mean hessian
+}
+
+type MetaMap struct {
```

Comment:
It seems like duplicated with protocol.CommonHeader

Reply:
fixed